override-json-file

```diff
@@ -0,0 +1 @@
+tests/resources/override.json
```

Comment:
@NatarajBTI I think you might need to symlink it the other way around, because if your docs is pointing to this file, they won't see the contents of the file. However if you added the override.json here and made `tests/resources/override.json` a symlink, then it should work

Reply:
Also in your last commit this file was in a directory, but now its in the root of the repo. Please confirm what your docs are going to point to (cc @Tejashree-Shiju )

Reply:
> @NatarajBTI I think you might need to symlink it the other way around, because if your docs is pointing to this file, they won't see the contents of the file. However if you added the override.json here and made `tests/resources/override.json` a symlink, then it should work

ok..will try to do other way but I am not sure whether that will work & whether symbolic link to file will be picked up in main.tf file here --> https://github.com/terraform-ibm-modules/terraform-ibm-mas/blob/main/tests/resources/main.tf

Reply:
> Also in your last commit this file was in a directory, but now its in the root of the repo. Please confirm what your docs are going to point to (cc @Tejashree-Shiju )

I am waiting for your approval & then will ask @Tejashree-Shiju to update the path to override.json file accordingly.